lib/utils/style-gather.html

```diff
@@ -8,6 +8,7 @@
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
 <link rel="import" href="resolve-url.html">
+<link rel="import" href="../elements/dom-module.html">
```

Comment:
This file shouldn't have an import of dom-module. The check below is to see if it's loaded and use it if so, but there are some uses of style-gather that don't necessitate loading dom-module, like custom-style.